syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -515,3 +517,60 @@ def get(self):
     def share(self, *owners, field=None, crypto_provider=None):
         self.child = self.child.share(*owners, field=field, crypto_provider=crypto_provider)
         return self
+
+    @staticmethod
+    def simplify(tensor: "FixedPrecisionTensor") -> tuple:
+        """Takes the attributes of a FixedPrecisionTensor and saves them in a tuple.
+
+        Args:
+            tensor: a FixedPrecisionTensor.
+
+        Returns:
+            tuple: a tuple holding the unique attributes of the fixed precision tensor.
+        """
+        chain = None
+        if hasattr(tensor, "child"):
+            chain = syft.serde._simplify(tensor.child)
+
+        return (
+            syft.serde._simplify(tensor.id),
```

Comment:
do we need to serialize these? they are strings/ints which msgpack handles fine

Reply:
Oops, indeed.

For ints I'm sure we don't need to simplify. But for strings we're using encoding something:

```
def _simplify_str(obj: str) -> tuple:
    return (obj.encode("utf-8"),)

```

Reply:
Given that said, I'll keep simplify for attributes that can be strings (potentially) if that's okay.

Reply:
sounds good to me